### PR TITLE
Alignment removes line breaks

### PR DESCRIPTION
**Pull request by dclause**
PR for issue #728

### CODE REVIEW

File: src/formats/bbcode.js [hunk 1]
```diff
@@ -775,6 +778,7 @@
 				]
 			},
 			isInline: false,
+      allowsEmpty: true,
```
Comment (brunoais) [inline src/formats/bbcode.js]: Tabs please.

Comment (dclause) [reply src/formats/bbcode.js]: Done

**Comment by brunoais** [comment]
Appears to work as expected sorry for taking so long

**Comment by dclause** [comment]
Happy to help :)

**Pull request merged**